packages/devtools/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: A suite of web-based performance tooling for Dart and Flutter.
 
 # Note: when updating this version, please update the corresponding entry in
 # lib/devtools.dart.
-version: 0.1.6-dev.2
+version: 0.1.7
```

Comment:
Are you increasing the version to release a new stable version? There is a polish bug with sliders we need to fix before releasing a stable version.
If you can use a dev release I would suggest that and we can release a new stable release in a week or two.

Reply:
Since we don't depend on this directly, I just need a version that published dwds can pull in.